README.md

```diff
@@ -35,6 +35,10 @@ It supports the following functionality:
 * Submit a changes to the commit-queue that includes changes from the local patch build and all
   enabled modules.
 
+## Documentation
+
+Read the documentation [here](https://evergreen-ci.github.io/evg-module-manager/).
```

Comment:
I already configured the github pages, so this link is live if you want to take a look.